lib/model/QubitInformationObject.php

```diff
@@ -2626,28 +2631,18 @@ public function importPhysicalObject($location, $name = false, $type = false, $l
       : false;
 
     // Check for an existing physical object within this collection with the same name
-    if ($this->parentId)
+    if ($options['collectionId'])
```

Comment:
I think the warning occurs when options is not initiated as a default array, otherwise in php `if (null)` is evaluated as false.